core/components/atoms/metricInput/MetricInput.tsx

```diff
@@ -307,23 +318,25 @@ export const MetricInput = React.forwardRef<HTMLInputElement, MetricInputProps>(
         </Text>
       )}
       {showActionButton && (
-        <div className={paginationStyles['MetricInput-arrowIcons']}>
-          <Button
+        <div className={arrowIconsClass}>
+          <button
             type="button"
-            icon="keyboard_arrow_up"
-            size={actionButtonSize}
-            className={`${actionButton} mb-2`}
+            className={arrowButtonClass}
```

Comment:
Instead of using `:first-child` selector in css file please directly use the `border` class here